### DISCUSSION TITLE
CRS error

### DISCUSSION THREAD

**Discussion by allenmoench** [earthlab-education/Earth-Analytics-AY24#580] [Q&A]
A new error I'm trying to figure out! Down at the bottom of the error message it says "CRSError: Invalid projection: EPSG:4326: (Internal Proj Error: proj_create: no database context specified)". 

![image](https://github.com/user-attachments/assets/3f208b43-fb7f-4dcc-9076-adf488a1ad1a)

This sounds like an issue with the Coordinate Reference System, but I'm not sure what exactly the problem is or how to fix it.

**Comment by lauren-pease** [comment]
Hi @allenmoench, I'm not certain if this is part of the problem, but I noticed on line 11 the if statement does not have an exists() method call because you have a space between exists and (redlining_path). 

Instead: 
`if not os.path.exists(redlining_path):`

**Reply by allenmoench** [reply]
Thanks! Elsa helped me sort this out - I  think it was an issue with how I had the environment set up.
